applications/asset_tracker_v2/src/modules/modules_common.c

```diff
@@ -84,7 +88,11 @@ int module_enqueue_msg(struct module_data *module, void *msg)
 			(struct event_type *)evt_proto->header.type_id;
 		char buf[50];
 
-		event->log_event(&evt_proto->header, buf, sizeof(buf));
+		if (event->log_event_func_dep) {
```

Comment:
We should also use ifdef here. I would also change order, because in general we should rely on the new implementation

Reply:
The same related to changes in line 50